<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>859</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>1024</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>10</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1024</width>
           <height>1024</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:10px;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0.994318, y2:0.148, stop:0 rgba(73, 134, 255, 255), stop:1 rgba(113, 255, 225, 255));</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>681</width>
            <height>799</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="2">
            <widget class="PlotWidget" name="widget_11" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_11">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(209, 209, 209);</string>
              </property>
              <property name="text">
               <string>Slice 11</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="PlotWidget" name="widget_13" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_14">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 13</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="PlotWidget" name="widget_12" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_12">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(209, 209, 209);</string>
              </property>
              <property name="text">
               <string>Slice 12</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="PlotWidget" name="widget_5" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 5</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="PlotWidget" name="widget_4" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 4</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="PlotWidget" name="widget_8" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_13">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 8</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="PlotWidget" name="widget_14" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_15">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 14</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="PlotWidget" name="widget_1" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>10</y>
                <width>43</width>
                <height>18</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string> Slice 1</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="PlotWidget" name="widget_3" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 3</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="PlotWidget" name="widget_7" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 7</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="4" column="0" colspan="4">
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSlider">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-radius:10px;</string>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
                <property name="indent">
                 <number>5</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="PlotWidget" name="widget_2" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 2</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="PlotWidget" name="widget_9" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 9</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="PlotWidget" name="widget_6" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>41</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 6</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="PlotWidget" name="widget_10" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_10">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 10</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="PlotWidget" name="widget_15" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_16">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 15</string>
              </property>
             </widget>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="PlotWidget" name="widget_16" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.086, x2:1, y2:1, stop:0 rgba(85, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
             </property>
             <widget class="QLabel" name="label_17">
              <property name="geometry">
               <rect>
                <x>60</x>
                <y>0</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(209, 209, 209);background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
              </property>
              <property name="text">
               <string>Slice 16</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0.994318, y2:0.148, stop:0 rgba(73, 134, 255, 255), stop:1 rgba(113, 255, 225, 255));</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad_Data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLoad_Data">
   <property name="text">
    <string>Load Data </string>
   </property>
  </action>
  <action name="actionLoad_model">
   <property name="text">
    <string>Load model</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
